public/app/plugins/datasource/loki/configuration/DerivedField.tsx

```diff
@@ -198,6 +202,22 @@ export const DerivedField = (props: Props) => {
           </Field>
         )}
       </div>
+
+      <div className="gf-form">
+        <Field label="Open in new tab" className={styles.openNewTab}>
```

Comment:
```suggestion
        <Field label="Open in a new tab" className={styles.openNewTab}>
```

Reply:
It's not a big deal, but I'm not 100% convinced. @jdbaldry what do you think?

Reply:
"Open a new tab" is more readable to me though "Open in new tab" is more common.

Can we centralize the label text somewhere that can be reused? I think that it would make sense to ensure that it's consistent.

I'd say that since that change is beyond the scope of this PR, we should stick with what's common and make the relabel a separate issue. It's a good first issue or a quick win for someone who can pick it up.

Reply:
Definitely. Thanks!